### DISCUSSION TITLE
Waiting for reply

### DISCUSSION THREAD

**Discussion by agnihackers** [bookwyrm-social/bookwyrm#2279] [General]
Hello @maintainer any update on this?

https://huntr.dev/bounties/00800781-52ad-4234-96d7-ee4552171495/

![Waiting](https://user-images.githubusercontent.com/58265761/185222901-74d209a1-d593-4186-b44c-0ec5ff2f6a34.jpg)

**Comment by mouse-reeve** [comment]
Hi, there's no need to open issues here as well, I see reports on huntr.dev. When I have updates, I'll post them there. While I get that it's frustrating for you to wait for a response, BookWyrm is a small, volunteer-run project and things happen when they happen.

**Reply by agnihackers** [reply]
Okay @mouse-reeve
